cis_v120/section_3.sp

```diff
@@ -32,6 +35,7 @@ control "cis_v120_3_1" {
     cis_item_id = "3.1"
     cis_level   = "2"
     cis_type    = "automated"
+    service     = "GCP/VPC"
```

Comment:
```suggestion
    service     = "GCP/Compute"
```

In AWS there's a VPC service and in Azure there's a Network service, but in the GCP API, is there a VPC or Network service? Are there any references to a service in any of our GCP plugins or mods for networking resources?